content/10-managing-databases/09-backup-considerations.mdx

```diff
@@ -0,0 +1,155 @@
+
+Replication won't safeguard your data as well as a backup would in a large number of failure scenarios.  The same mechanism that ensures that all changes are copied to a secondary server will also duplicate any problems with your primary database.  For instance, if a record is unintentionally deleted on the primary database, that change will also be executed on any downstream replicas.  This same process means that corrupt data will also be disseminated.
+
+Another reason replication is not a backup is that it doesn't have any obvious ability to restore data.  While you can replicate changes back and forth between servers, replication isn't concerned with keeping previous versions of data and any historic view of your data will likely be lost when the logs are rotated.  This "deficiency" is a reminder that replication is primarily designed to help organizations increase availability and performance rather than as a data preservation tool.
+
+With that being said, replication can be an important component of a backup and disaster recovery plan.  For instance, replication can be useful in the event of a hardware failure on the primary database.  In this case, administrators can quickly promote a replication follower to the leader role and continue to serve client requests to avoid a lengthy restoration procedure.  Some organizations also set up a "delayed" replica which only applies changes after a period of time has elapsed, allowing them to quickly "roll back" the database state by switching to the replica if necessary.
+
+Another case where replication is often involved in backup processes is as the target of backup operations.  Many times, it's easiest to back up a replica than the primary server.  For instance, to get a consistent file-level backup, you could configure a secondary database as a replica of your production database.  Once the database is synchronized, you can turn off replication temporarily and perform a backup of the replica without affecting your production traffic.  When the backup is complete, you can turn replication back on to resynchronize the server with the changes that occurred during the backup window.
+
+## How do backups affect security?
+
+Backup strategy intersects with security considerations in a few different ways.
+
+Backups can help protect against certain security incidents by providing secondary sources of data in scenarios like ransomware attacks.  For instance, if an intruder is able to access and encrypt the contents of your primary database, having access to historical snapshots may give you more options for addressing the situation.  For this to be an option, your backup destination be in a separate security context than your source data to prevent an attacker from impacting your backups as well.
```

Comment:
```suggestion
Backups can help protect against certain security incidents by providing secondary sources of data in scenarios like ransomware attacks.  For instance, if an intruder is able to access and encrypt the contents of your primary database, having access to historical snapshots may give you more options for addressing the situation.  For this to be an option, your backup destination must be in a separate security context than your source data to prevent an attacker from impacting your backups as well.
```